src/pages/guides/rust/metaplex-rust-sdks.md

```diff
@@ -0,0 +1,197 @@
+
+```rust
+mpl_core::types
+```
+
+### Errors
+
+While an **errors** module is generated for every SDK this just holds the error list for that specific program and users do not need to interact with this module.
+
+
+## Instruction Builders
+
+Metaplex Rust SDKs will also currently come with two a **Builder** versions of each instruction which you can import. This abstracts a massive amount code for you and will return you an instruction that's ready to send.
+
+These include:
+
+- Builder
+- CpiBuilder
+
+In the case of `CreateV1` from the [mpl-Core crate docs](https://docs.rs/mpl-core/0.7.0/mpl_core/instructions/index.html) these instructions are currently available to us.
+
+```
+CreateV1
+CreateV1Builder
+CreateV1Cpi
+CreateV1CpiAccounts
+CreateV1CpiBuilder
+CreateV1InstructionArgs
+CreateV1InstructionData
+```
+
+Each instruction that comes from a Metaplex Rust crate 
+
+Lets take the `CreateV1` instruction from Core as an example (this applies to all other instructions from this Crate and all other Metaplex crates too).
+
+If we look through the instructions in the  we can see we have a number of instructions available to us.
+
+### Builder
+
+Builder instructions are designed to be used via
+
+The one we are interested in here is the `CreateV1Builder`.
+
+To initialize the builder we can call `new`.
+
+```rust
+CreateV1Builder::new();
+```
+
+From this point we can `ctrl + click` (pc) or `cmd + click` (mac) into the `new` function generated from the `Builder::` which positions us at the `pub fn new()` for the builder. If you scroll up slightly you'll then see the `pub struct` for the `CreateV1Builder` as outlined below.
+
+```rust
+pub struct CreateV1Builder {
+    asset: Option<solana_program::pubkey::Pubkey>,
+    collection: Option<solana_program::pubkey::Pubkey>,
+    authority: Option<solana_program::pubkey::Pubkey>,
+    payer: Option<solana_program::pubkey::Pubkey>,
+    owner: Option<solana_program::pubkey::Pubkey>,
+    update_authority: Option<solana_program::pubkey::Pubkey>,
+    system_program: Option<solana_program::pubkey::Pubkey>,
+    log_wrapper: Option<solana_program::pubkey::Pubkey>,
+    data_state: Option<DataState>,
+    name: Option<String>,
+    uri: Option<String>,
+    plugins: Option<Vec<PluginAuthorityPair>>,
+    __remaining_accounts: Vec<solana_program::instruction::AccountMeta>,
+}
+
+```
+
+These are your arguments of publickeys and data that will need to be passed into the builder. Some accounts may also be optional and default to others, this can vary from instruction to instruction. If you click through to the `new()` function again and scroll down this time you'll see the individual functions with additional comments. In the below case you can see that the owner will default to payer, so we don't need to pass in owner if in this case if the payer is also going to be the owner of the Asset.
```

Comment:
Phrasing is slightly unclear: "Some accounts may also be optional and default to others"

Recommend reword for clarity.

Reply:
`Some accounts may also be optional. These optional accounts my not be required at all by the program or could possibly default to another address if left out. This behaviour can vary from instruction to instruction. `